docs/jwst/refpix/description.rst

```diff
@@ -107,6 +107,9 @@ and/or rotated to convert to the detector frame.
 NIR Data
 ++++++++
 
+For single amplifier readout (NOUTPUTS keyword = 1):
+----------------------------------------------------
```

Comment:
Might need a different section indicator for these. These should be a sub-section under "NIR Data", but it's using the same rst section markup as the higher level section "Subarrays". Perhaps "~~~~~~~" will work?